src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -551,8 +551,6 @@ public PaymentTransactionInfoPlugin authorizePayment(final UUID kbAccountId, fin
                                     rbacUsername,
                                     rbacPassword
                             ));
-                } else if (PaymentServiceProviderResult.REDIRECT_SHOPPER.getResponses()[0].equals(responsesRecord.getResultCode())) {
```

Comment:
Did we had a discussion and confirmation on not adding the time out handling for Klarna Redirect Shopper ?

Reply:
Yes. This will be handled as other in complete auth transactions which are redirected but don't get completed for any reason. PENDING AUTH transactions will be marked as PLUGIN_FAILURE by Janitor after timeout (1H).